src/main/webapp/WEB-INF/web.xml

```diff
@@ -10,7 +10,7 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>GenericServlet</servlet-name>
-        <url-pattern>/</url-pattern>
+        <url-pattern>/index</url-pattern>
```

Comment:
In the end, this was the fix.

Resolution is in part 2 of the answer.
https://stackoverflow.com/questions/29011814/jsp-doesnt-see-css

Give it a go now @Tom-Goring . It's working on my end again.

Reply:
And it works! Nice find!
![image](https://user-images.githubusercontent.com/26003386/101492192-8ff67400-395c-11eb-9df2-235033f12eca.png)